Winter 2019 Fanart Contest newspost (#2807)
ppy/osu-wiki: news/2019-11-11-winter-2019-fanart-contest.md

---
layout: post
title: "Winter 2019 Fanart Contest"
date: 2019-11-11 18:00:00 +0000
---

The end of the year is rapidly approaching, and while things might cooling down where you are (or maybe not), the artist's battlefield is still just as firey as ever!

![](https://assets.ppy.sh/contests/82/header.jpg)

Do you like to draw? Great, because we've got just the contest for you!

> **Draw any of the osu! mascots (official or community) in a winter themed piece** and you could have a shot at getting your work showcased in the game's main menu as a part of our Winter 2019 Seasonal Background pack!

Reviewer: ```suggestion
**Draw any of the osu! mascots (official or community) in a winter-themed piece** and you'll have a shot at getting your work showcased in the game's main menu as a part of our Winter 2019 Seasonal Background pack!
```

> The hard-fought finalists will also nab themselves a few months of **osu!supporter** (the exact amount yet to be determined), and the top three finalists as voted by YOU will receive an extra shiny profile badge showcasing their work!

Reviewer: ```suggestion
The hard-fought finalists will also nab themselves a few months of **osu!supporter** (the exact amount yet to be determined), and the top three finalists as voted by YOU will receive **an extra shiny profile badge** showcasing their work!
```

> But Ephemeral, you might ask, these are *new* prizes, what gives? What gives is the season of giving, and that's precisely what this time of year tends to be for most of the world. That and most importantly, we've heard your feedback on the prizes for the winning artists crystal clear. *(Yes, previous fanart contest top 3 finalists will eventually receive profile badges for their winning entries as well. You'll not be left out!)*

Reviewer: ```suggestion
"But Ephemeral," you might ask, "these are *new* prizes, what gives?" What gives is the season of giving, and that's precisely what this time of year tends to be for most of the world. That and most importantly, we've heard your feedback on the prizes for the winning artists crystal clear. *(Yes, previous fanart contest top 3 finalists will eventually receive profile badges for their winning entries as well. You won't be left out!)*
```
interesting how 3 words can be contracted in different ways

Think you're up for the challenge? You've got **20** days from RIGHT NOW to get your concept onto paper and into your computer, then in to our contest system. As always, the community will be forced to make many impossible choices with far too many votes to help us decide which ones will be our final entries.

> Make sure you read the rules below before you enter - especially if you've done this before, since there's a few new ones this time around!

Reviewer: ```suggestion
Make sure you read the rules below before you enter, even if you've done this before, since there's a few new ones this time around!
```

## Rules

> * **All entries MUST be winter themed and feature one (or more) of the osu! mascots (either official or community).** Cold, Christmas, mountains or whatever else, just make sure that it is really obviously winter at a glance for most of the world and that a mascot we know and love features in it. Any lonesome Dons will find themselves figuratively and literally out in the cold this year.

Reviewer: ```suggestion
* **All entries MUST be winter themed and feature one (or more) of the osu! mascots (either official or community).** Cold, Christmas, mountains or whatever else, just make sure that it is really obviously winter at a glance for most of the world and that a mascot we know and love is featured in it. Any lonesome Dons will find themselves figuratively and literally out in the cold this year.
```
"features" is used when the object is being featured, but in this case you're talking about subject

> * **All entries must be suitable for all-ages viewing.** Our mascots are pure and innocent and would never be wrapped up in nothing but ribbons and put under a festive tree. Someone might have done this a few years back and we had to pull their entry because of it - YOU KNOW WHO YOU ARE! Keep it clean.

Reviewer: ```suggestion
* **All entries must be suitable for all-ages viewing.** Our mascots are pure and innocent and would never be wrapped up in nothing but ribbons and put under a festive tree. Someone might have done this a few years back and we had to pull their entry because of it — YOU KNOW WHO YOU ARE! Keep it clean.
```
grand compromise

* **DO NOT LEAVE THE DOTTED osu! COOKIE TEMPLATE VISIBLE IN YOUR SUBMISSION!** Please don't do this. It doesn't need to be there when you're done drawing, it just makes your work look worse.
* **ACCOUNT FOR THE DOTTED osu! COOKIE TEMPLATE IN THE COMPOSITION OF YOUR WORK!** We've already said this, but anything inside the dotted circle will *not* be seen during voting phase and will be very difficult to see on the main menu. You still need to draw something there — don't just leave it blank, but be creative about it. Check out previous contest entries for inspiration!
* **MAKE SURE YOU DON'T HAVE THE DOTTED osu! COOKIE TEMPLATE VISIBLE IN YOUR WORK!** What's this, the same rule repeated three times in three slightly different ways? What could this POSSIBLY MEAN? If you're not following, it means to TRIPLE CHECK that you've not left the template visible in your work. 

> * **Entries MUST meet the resolution requirements for a main-menu piece - namely 2732x1536. THIS IS CHECKED BY A SCRIPT AND ENTRIES THAT DO NOT MATCH THIS REQUIREMENT WILL BE DISCARDED AUTOMATICALLY.** Just in case you missed it, a script automatically discards entries that don't meet the resolution requirements, even if they're off by only a few pixels. This is the #1 reason why people don't see their artwork in the end voting, and many excellent entries have met this tragic, tragic fate. Export your entry at the proper size!

Reviewer: ```suggestion
* **Entries MUST meet the resolution requirements for a main-menu piece — namely 2732x1536. THIS IS CHECKED BY A SCRIPT AND ENTRIES THAT DO NOT MATCH THIS REQUIREMENT WILL BE DISCARDED AUTOMATICALLY.** Just in case you missed it, a script automatically discards entries that don't meet the resolution requirements, even if they're off by only a few pixels. This is the #1 reason why people don't see their artwork in the end voting, and many excellent entries have met this tragic, tragic fate. Export your entry at the proper size!
```

> * **SUBMIT ONLY YOUR OWN WORK! NO TRACING! NO REDRAWING! NO COMPOSITE WORKS!** Just don't plagarize. Don't rip off doujin/manga stills, don't redraw characters, don't do anything that isn't your own work and creativity taking the fore. This contest is about YOUR skill, not how well you can pass off someone else's work at your own. You will be permanently banned from future Fan Art contests if you are caught doing this!

Reviewer: ```suggestion
* **SUBMIT ONLY YOUR OWN WORK! NO TRACING! NO REDRAWING! NO COMPOSITE WORKS!** Just don't plagarize. Don't rip off doujin/manga stills, don't redraw characters, don't do anything that isn't your own work and creativity taking the fore. This contest is about YOUR skill, not how well you can pass off someone else's work at your own. You will be permanently banned from future fanart contests if you are caught doing this!
```
